opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -18,26 +18,33 @@
 /**
  * Tracer is a simple, thin interface for Span creation and propagation across arbitrary transports.
  */
-public interface Tracer {
+public interface Tracer extends ActiveSpanSource {
```

Comment:
@pavolloffay it's always fine to start a new Span via `SpanBuilder.startActive()` regardless of whether there's an `activeSpan()` – not sure if I understand your meaning.

I think `adopt(Span)` is indeed hard to find, but it's also something I don't expect people to use very often. The try-with-resources pattern is a definite best practice (since otherwise it's hard to remember to deal with exceptions / `finally` clauses) and I'm imagining that most straight-line Span code happens within a `try {}` block.

I have deprecated `start()` and replaced it with the non-`ActiveSpan`-activating `Span startManual()`.